Frameworks/UIKit/NSString+UIKitAdditions.mm

```diff
@@ -263,7 +263,14 @@ - (CGSize)_sizeWithAttributes:(NSDictionary<NSString*, id>*)attributes constrain
         size.height = std::numeric_limits<CGFloat>::max();
     }
 
-    return CTFramesetterSuggestFrameSizeWithConstraints(framesetter.get(), CFRangeMake(0, self.length), nullptr, size, nullptr);
+    CGSize ret = CTFramesetterSuggestFrameSizeWithConstraints(framesetter.get(), CFRangeMake(0, self.length), nullptr, size, nullptr);
+
+    // If the constrained height is less than a line height sizeWithFont will increase the returned size to fit at least one line
+    UIFont* font = attributes[NSFontAttributeName];
```

Comment:
Looks like reference platform just returns CGSizeZero, will update to reflect that